test/gtest/uct/uct_test.cc

```diff
@@ -147,6 +147,10 @@ void uct_test::set_sockaddr_resources(uct_md_h md, char *md_name, cpu_set_t loca
 
 std::vector<const resource*> uct_test::enum_resources(const std::string& tl_name,
                                                       bool loopback) {
+    // Run the tests for all devices for TCP by default
+    static char *env = getenv("GTEST_UCT_TCP_DEVS");
```

Comment:
according to name it might be a list of devices but it's handled like specific value

Reply:
fixed